plugins/modules/packaging/os/homebrew.py

```diff
@@ -532,10 +541,13 @@ def _upgrade_all(self):
             self.changed = True
             self.message = 'Homebrew packages would be upgraded.'
             raise HomebrewException(self.message)
-        rc, out, err = self.module.run_command([
-            self.brew_path,
-            'upgrade',
-        ])
+        opts = (
+            [self.brew_path, 'upgrade']
+            + self.upgrade_options
```

Comment:
I would propose to init upgrade_options to an empty list similar how it is done for install_options (line 367)

Reply:
Done.